### ISSUE TITLE
Availability on NPM and Bower

### ISSUE THREAD

**Issue by dkrutsko** [robflaherty/riveted#28]
Hey there, not sure if this has already been asked but would you be willing to publish this library on NPM and Bower. I noticed that you already have a package and bower file but at least for NPM, I haven't seen an up to date version of this library on there.

**Comment by robflaherty** [comment]
Hey, I would love to but someone else registered the Riveted name on npm. So I don't control it. I asked them in the past if they would transfer ownership to me. I'll try asking again!

**Comment by dkrutsko** [comment]
You may have success with contacting NPM support directly. I had a similar problem with one of my own projects and support was kind enough to transfer ownership to me. This should be an easy case since the version number has not been updated and the last publish was over two years ago. In fact there's like two versions of riveted up there, both by the same guy, no idea why though. I wish you luck though!

**Comment by greggb** [comment]
Found this module. Is it an official fork?
https://www.npmjs.com/package/riveted-node
https://github.com/tamarasaurus/riveted
https://github.com/robflaherty/riveted/compare/master...tamarasaurus:master

**Comment by robflaherty** [comment]
@greggb no, someone else claimed the package name and i haven't been able to get ownership of it

**Comment by greggb** [comment]
Ah, bummer. I kind of figured that and was worried it might be malicious - looks like it's strictly to get it on NPM though, luckily. She seems fairly active on GH, so it may be worth a shot to ask about transferring.
